src/sagas/ui.js

```diff
@@ -25,6 +28,20 @@ export function* projectSuccessfullySaved() {
   yield put(hideSaveIndicator());
 }
 
+export function* assignmentCreated({payload: {assignment}}) {
+  if (assignment.alternateLink) {
+    yield put(assignmentDisplayed({
+      url: assignment.alternateLink,
+      exportType: 'assignment',
+    }));
+  } else {
+    yield put(assignmentDisplayed({
+      url: 'https://classroom.google.com/',
+      exportType: 'assignment-draft',
+    }));
```

Comment:
It seems like we’re just consuming an action and re-dispatching a new action? Why not just have the reducers that are currently consuming `ASSIGNMENT_DISPLAYED` instead consume `ASSIGNMENT_CREATED` directly?